src/main/java/com/esri/samples/map/create_and_save_map/README.md

```diff
@@ -0,0 +1,39 @@
+<h1>Create and Save Map</h1>
+
+<p>How to create and save a map to your own portal.</p>
+
+<p><img src="CreateAndSaveMap.png"/></p>
```

Comment:
Like this better if a gif. Could start like below and end with saving.
![login3](https://cloud.githubusercontent.com/assets/14575538/21046351/36183976-bdfc-11e6-80bc-97c1e4680d65.PNG)
![login2](https://cloud.githubusercontent.com/assets/14575538/21046354/39b32fc8-bdfc-11e6-96dd-3133970a02b5.PNG)